core/src/main/java/com/netease/arctic/table/BaseUnkeyedTable.java

```diff
@@ -101,7 +101,7 @@ public void refresh() {
 
   @Override
   public TableScan newScan() {
-    return icebergTable.newScan().includeColumnStats();
+    return icebergTable.newScan();
```

Comment:
Does Iceberg's TableScan not `includeColumnStats` as default?

Reply:
yes